Description: Fix FTBFS with new glib-2.0
Origin: http://code.google.com/p/gimmix/source/detail?r=3
Bug-Debian: http://bugs.debian.org/741790

Index: gimmix/src/gimmix-playlist.c
===================================================================
--- gimmix.orig/src/gimmix-playlist.c	2014-03-20 11:21:33.543082329 +0100
+++ gimmix/src/gimmix-playlist.c	2014-03-20 11:23:11.309823346 +0100
@@ -39,7 +39,7 @@
 
 typedef enum {
 	SONG = 1,
-	DIR,
+	GIMMIX_DIR,
 	PLAYLIST
 } GimmixFileType;
 
@@ -149,7 +149,7 @@
 		gtk_tree_model_get_iter (model, &iter, list->data);
 		gtk_tree_model_get (model, &iter, 2, &path, 3, &type, -1);
 		
-		if (type == DIR || type == SONG)
+		if (type == GIMMIX_DIR || type == SONG)
 		{
 			switch (target_type)
 			{
@@ -696,7 +696,7 @@
 						0, dir_pixbuf,
 						1, path,
 						2, data->directory,
-						3, DIR,
+						3, GIMMIX_DIR,
 						-1);
 			g_free (path);
 		}
@@ -954,7 +954,7 @@
 		gtk_tree_model_get_iter (model, &iter, list->data);
 		gtk_tree_model_get (model, &iter, 2, &path, 3, &type, -1);
 		
-		if (type == DIR)
+		if (type == GIMMIX_DIR)
 		{	
 			gimmix_update_library_with_dir (path);
 		}
@@ -1005,7 +1005,7 @@
 		gtk_tree_model_get_iter (model, &iter, list->data);
 		gtk_tree_model_get (model, &iter, 2, &path, 3, &type, -1);
 		
-		if (type == DIR)
+		if (type == GIMMIX_DIR)
 		{
 			mpd_playlist_queue_add (gmo, path);
 		}
@@ -1023,7 +1023,7 @@
 		gtk_tree_model_get (model, &iter, 2, &path, 3, &type, -1);
 	
 		
-		if (type == DIR)
+		if (type == GIMMIX_DIR)
 		{	
 			mpd_playlist_queue_add (gmo, path);
 			g_free (path);
@@ -1081,7 +1081,7 @@
 		gtk_tree_model_get_iter (model, &iter, list->data);
 		gtk_tree_model_get (model, &iter, 2, &path, 3, &type, -1);
 		
-		if (type == DIR)
+		if (type == GIMMIX_DIR)
 		{
 			mpd_playlist_queue_add (gmo, path);
 		}
@@ -1099,7 +1099,7 @@
 		gtk_tree_model_get (model, &iter, 2, &path, 3, &type, -1);
 	
 		
-		if (type == DIR)
+		if (type == GIMMIX_DIR)
 		{	
 			mpd_playlist_queue_add (gmo, path);
 			g_free (path);
@@ -1245,7 +1245,7 @@
 					0, dir_pixbuf,
 					1, "..",
 					2, parent,
-					3, DIR,
+					3, GIMMIX_DIR,
 					-1);
 		g_free (parent);
 	}
@@ -1260,7 +1260,7 @@
 								0, dir_pixbuf,
 								1, directory,
 								2, data->directory,
-								3, DIR,
+								3, GIMMIX_DIR,
 								-1);
 			g_free (directory);
 		}
@@ -1341,7 +1341,7 @@
 	gtk_tree_model_get_iter (model, &iter, list->data);
 	gtk_tree_model_get (model, &iter, 2, &path, 3, &type, 4, &id, -1);
 
-	if (type == DIR)
+	if (type == GIMMIX_DIR)
 	{
 		g_free (path);
 		return;
